src/qz/utils/SystemUtilities.java

```diff
@@ -264,10 +263,13 @@ public static boolean isDarkMode(boolean recheck) {
     }
 
     public static void adjustThemeColors() {
-        if (isDarkMode()) {
-            Constants.WARNING_COLOR = Constants.WARNING_COLOR_LIGHTER;
-            Constants.TRUSTED_COLOR = Constants.TRUSTED_COLOR_LIGHTER;
-        }
+        Constants.WARNING_COLOR = isDarkMode() ? Constants.WARNING_COLOR_DARK : Constants.WARNING_COLOR_LITE;
+        Constants.TRUSTED_COLOR = isDarkMode() ? Constants.TRUSTED_COLOR_DARK : Constants.TRUSTED_COLOR_LITE;
+    }
+
+    public static boolean prefersMaskTrayIcon() {
+        // TODO: Test on Windows
```

Comment:
Is this todo something meant to be targeted in this PR?

Reply:
@bberenz may be left in the codebase.  Windows too has monochrome tray icons, but the theming of the start bar is different (some light mode installs have a dark bar).  I'd like to support this but may need some platform-specific code to know which color that icon should be.  If you can find information on this, I'd be happy to add it to this PR.

Reply:
On Windows 10 Pro, I can't seem to trigger dark tray icons at all.

Here's me trying to manipulate the taskbar to be white.  Notice when the color is darker than a particular threshold Windows says `This color is not supported`.
![image](https://user-images.githubusercontent.com/6345473/61824334-baa96080-ae2b-11e9-9826-8ff915b55732.png)

Rightfully so, here's what white looks like:
![image](https://user-images.githubusercontent.com/6345473/61824381-d876c580-ae2b-11e9-88a2-403b3441a822.png)

What puzzles me is I remember my HP Spectre laptop having a light themed start bar on install.  It was Windows 10 Home (not professional) but it's since been reloaded with professional. I may have to setup a VM to fully test this.

Reply:
Light Mode for Windows 10 appears to be brand-new with Cumulative Update 1903, which is opt-in currently.  Installing now in a VM to see what it's like.

On a side-note -- looking at Server 2012 -- it appears the Windows icons are white with a dark border/drop-shadow.

We may be able to add this same drop-shadow programmatically.  Pinging @Vzor- for insight. :)

> ![image](https://user-images.githubusercontent.com/6345473/61830759-fea36200-ae39-11e9-95f6-45f3e52b506f.png)

Reply:
It's fine if its not, just wanted to make sure it wasn't overlooked.
I wasn't aware if windows had any icon masking at all. Interesting to know it has better light color support coming, that one has always annoyed me. 
Based on this I think it might be best to come back to it later then when window's has wider support for it released.

Reply:
Added via 860e559, but only for Windows 10 and higher.  Windows 7 seemed to use a stroke around the white icons to avoid some issues with lighter themes.  Preview added below.